plugin/web/components/sidebar/index.tsx

```diff
@@ -1,15 +1,11 @@
-import { Button } from "antd";
-
-import Wrapper from "../common/Wrapper";
+import MainLayout from "./Layout/MainLayout";
+import "antd/dist/antd.less";
+import "../../theme/styles/theme.less";
 
 const Sidebar: React.FC = () => {
   return (
     <>
-      {/* SIDEBAR CONTENTS GO HERE */}
-      {/* BELOW WRAPPER IS JUST AN EXAMPLE */}
-      <Wrapper>
-        <Button type="primary">Primary Button</Button>
-      </Wrapper>
+      <MainLayout isInsideEditor={false}></MainLayout>
```

Comment:
Please delete the fragment surrounding this component. They aren't needed any more

Reply:
Also, please write this component like `<MainLayout isInsideEditor={false} />`

Reply:
Done

Reply:
https://github.com/eukarya-inc/reearth-plateauview/pull/29/commits/326aa3062b26c0e076d4cc56cf8ad8237c0cf1e2